charts/README.md

```diff
@@ -0,0 +1,13 @@
+
+# Kubernetes Test-Infra Helm charts
+
+This is the [helm](https://github.com/kubernetes/helm) chart repository for test infra components. Helm is a package manager for kubernetes  this chart repository adapts test-infra components for general use on any kubernetes cluster. Please see documentation on individual components for more detail.
+
+To use this repository to install prow:
+
+```bash
+git clone https://github.com/kubernetes/test-infra.git
+cd charts/prow
+helm install . --set repos={"github_org/repo1" "github_org/repo2" ...}
```

Comment:
Those are the repos that prow will manage.  The way the chart works is it takes a list of repos and generates hoks and configs automagically.